src/Files.FullTrust/MessageHandlers/FileOperationsHandler.cs

```diff
@@ -35,15 +35,12 @@ public void Initialize(PipeStream connection)
             progressHandler = new ProgressHandler(connection);
         }
 
-        public async Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, JsonElement> message, string arguments)
-        {
-            switch (arguments)
+        public Task ParseArgumentsAsync(PipeStream connection, Dictionary<string, object> message, string arguments)
```

Comment:
I think this line may have some merging issue. The message should be `Dictionary<string, JsonElement>`.

Reply:
Oh, I think that was why pipeline builds were failing.